recipe/conda_build_config.yaml

```diff
@@ -442,7 +442,8 @@ libcurl:
 libdap4:
   - 3.19
 libevent:
-  - 2.0.22
+  - 2.0.22   # [not (aarch64 or ppc64le)]
+  - 2.1.10   # [aarch64 or ppc64le]
```

Comment:
For the sake of simplicity let's migrate all the stack to the new `libevent 2.1.10`. We are bound to update at some point anyways.

Reply:
Changed it and made a migrator: https://github.com/regro/cf-scripts/pull/521